### ISSUE TITLE
Add inline tests

### ISSUE THREAD

**Issue by Algor0113** [MetaMorphic-Digital/torgeternity#89]
Hey Matt,

we've talked about it: please embed a function to request roll tests in chats and in journal entries with a chat/journal-command.

Thank you a lot :) (Enhancement)

**Comment by Algor0113** [comment]
As I've just learned, they're called "inline tests"

**Comment by gmmatt** [comment]
Sounds great. Would really love to do this!

**Comment by pemerson71** [comment]
Alright, this says the issue is open, so I'm going to pipe in, here.  It's a great idea to do inline rolls; I know for a fact they work in The One Ring by Herve Derritchon and I think they would be an amazing addition to our game system.

**Comment by Algor0113** [comment]
Postponed after #326

**Comment by farling42** [comment]
Version 13.1.0 should support inline checks of the form:

```
@Check[persuasion|dn=18]
@Check[taunt|dn=standard]{Taunt}
@Check[interactionAttack|skillName=intimidation|dn=targetIntimidation|unskilledUse=true|interactionAttackType=intimidation]
```

The third example is showing that all parameters to the TestDialog can be specified as part of the macro.

The inline links can be triggered directly from the journal/actor/item, or the GM can post the link to chat for players to make the roll from that link.

(All original code, but loosely based on the syntax used by the PF2E game system.)

**Comment by Algor0113** [comment]
Completed with release v13.1.0
